### ISSUE TITLE
Passing environment variables through to the task

### ISSUE THREAD

**Issue by porty** [buildkite/ecs-run-task#11]
It would be nice to be able to pass through environment variables to the task to be run. I would propose a new command-line flag to do the task:

```shell
$ ecs-run-task \
    --file my-task-definition.yml \
    --cluster my-cluster \
    --env KEY=VALUE \
    --env KEY_FROM_HOST \
    my-command-to-run
```

I'm happy to take point on implementation, but I'd like to know if this is a viable feature request and also how to specify the environment variables.

As per the example I would have the flag `--env` or `-e` for short with a key/value pair separated by `=`. If no value is specified it is taken from the existing environment similar to [what is done in `docker-compose.yml` environment sections](https://docs.docker.com/compose/compose-file/#environment).

What I'm not proposing:
* Passing through the whole environment of the running process
  * The environment that `ecs-run-task` is running in has little or nothing to do with the remote task
* ~Passing through a single current environment variable~
  * ~Could be a nice shorthand but I don't need this right now~

Thoughts?

**Comment by lox** [comment]
Sounds legit, would happily accept it. IMO passing through a current env would be very useful, otherwise it's a good way to leak secrets in env values logs and to Buildkite.

**Comment by porty** [comment]
Sounds good, I'll include them (description updated)

**Comment by porty** [comment]
Closed by #12
